src/name_bazaar/ui/pages/user_offerings_page.cljs

```diff
@@ -71,7 +71,9 @@
               :tablet 8
               :mobile 16
               :floated "right"}
-             [share-buttons]]]
+             [share-buttons
+              @(subscribe [:page-share-url :route.user/offerings {}])
+              "My ENS Offerings"]]]
```

Comment:
Actually, users can share other user's offerings page as well. So here should detect if it shares my offerings or a user offerings page and choose title accordingly.